basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,122 @@
 const helpers = require('./project-1');
 
-// start testing!
+
+describe('Jest', ()=> {
+  test('Jest works', () => {
+
+  })
+})
+
+describe('The tests in this here thing', () => {
+
+  it('multiplys by ten', ()=> {
+    const result = helpers.multiplyByTen(2)
+    expect(result).toBe(20)
```

Comment:
You should also test for the failure cases. For example what it should NOT be. What happens if instead of number You pass a string or invalid object etc.
Testing is about catching all those small errors and things that can happen while software is running.